coreplugins/cesiumion/public/components/TasksDialog.jsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import ErrorMessage from './ErrorMessage';
+import IonAssetLabel from './IonAssetLabel';
+import './TaskDialog.scss';
+import $ from 'jquery';
```

Comment:
Ewww jquery. Do other places in the web odm repo reference jquery in React? Was this leftover from the existing implementation?

Reply:
used in other places